harness/determined/cli/task.py

```diff
@@ -166,6 +166,20 @@ def kill(args: Namespace) -> None:
     print(f"Sucessfully killed task: {args.task_id}")
 
 
+@authentication.required
+def pause(args: Namespace) -> None:
+    sess = cli.setup_session(args)
+    bindings.post_PauseGenericTask(sess, taskId=args.task_id)
+    print(f"Sucessfully paused task: {args.task_id}")
+
+
+@authentication.required
+def resume(args: Namespace) -> None:
+    sess = cli.setup_session(args)
+    bindings.post_ResumeGenericTask(sess, taskId=args.task_id, projectId=args.project_id)
```

Comment:
got 
```
File "/Users/ilia/repo/determined/harness/determined/cli/task.py", line 179, in resume
    bindings.post_ResumeGenericTask(sess, taskId=args.task_id, projectId=args.project_id)
TypeError: post_ResumeGenericTask() got an unexpected keyword argument 'projectId'
```

here. am I missing something?

(removing this arg fixes it, but I have not tested the workspaces/projects aspects)

Reply:
Forgot to update the bindings call. For some reason it was still working.